src/components/SemesterPicker/SemesterPicker.tsx

```diff
@@ -70,9 +70,9 @@ export const SemesterPicker: FC<{page: 'zadania' | 'vysledky' | 'admin/opravovan
   }
 
   return (
-    <div className={styles.menu}>
+    <Stack sx={{flexDirection: 'row', alignItems: 'center', gap: 2, zIndex: 1000, userSelect: 'none'}}>
```

Comment:
idealne `Stack direction="row"` a nesahat na `flexDirection` priamo

Reply:
ja som akosi vsetko natrepal do sx, lebo som mal pocit ze to tak chceme :-D